src/site/elements/input.overrides

```diff
@@ -1,3 +1,9 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.ui.disabled.input input,
+.ui.form .ui.disabled.input input,
+.ui.input:not(.disabled) input[disabled] {
+  background-color: grey
```

Comment:
hmm or should we aim to have all values in `variables`?  e.g. `@disabledInputBackground`